clients/errors.go

```diff
@@ -6,4 +6,5 @@ var (
 	ErrPasswordIsTooShort = errors.New("password is too short")
 	ErrMustSpecifyOrg     = errors.New("must specify org ID or org name")
 	ErrMustSpecifyBucket  = errors.New("must specify bucket ID or bucket name")
+	ErrMustSpecifyStack   = errors.New("must specify stack ID")
```

Comment:
Since this is only used by `export`, I'd put it in `export.go` for now. The other errors here cut across multiple commands.